### PR TITLE
NoShortEchoTagFixer - adjust isCandidate check for hhvm

### PR DESCRIPTION
**Pull request by keradus**
hhvm on 2.x line is broken :( let us fix it !

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:+1: :ship:

File: src/Fixer/PhpTag/NoShortEchoTagFixer.php [hunk 1]
```diff
@@ -26,7 +26,17 @@
      */
     public function isCandidate(Tokens $tokens)
     {
-        return $tokens->isTokenKindFound(T_OPEN_TAG_WITH_ECHO);
+        return $tokens->isTokenKindFound(T_OPEN_TAG_WITH_ECHO)
+            /*
+             * HHVM parses '<?=' as T_ECHO instead of T_OPEN_TAG_WITH_ECHO
+             *
+             * @see https://github.com/facebook/hhvm/issues/4809
```
Comment (GrahamCampbell) [inline src/Fixer/PhpTag/NoShortEchoTagFixer.php]: You can't use phpdoc annotations when this isn't `T_DOC_COMMENT`.

Comment (SpacePossum) [inline src/Fixer/PhpTag/NoShortEchoTagFixer.php]: >You can't use phpdoc annotations when this isn't T_DOC_COMMENT.

there are no rules for comment content we follow, there is no need also, this PR proves that

Comment (keradus) [inline src/Fixer/PhpTag/NoShortEchoTagFixer.php]: I could use anything inside regular comment...

**Pull request closed without merging**